packages/cli/src/runner/crawler-command-runner.spec.ts

```diff
@@ -3,79 +3,148 @@
 import 'reflect-metadata';
 
 import { CrawlerEntryPoint, CrawlerRunOptions, ScanResults } from 'accessibility-insights-crawler';
-import { IMock, Mock, Times } from 'typemoq';
+import * as fs from 'fs';
+import { IMock, It, Mock, Times } from 'typemoq';
 import { ReportDiskWriter } from '../report/report-disk-writer';
 import { ReportGenerator } from '../report/report-generator';
 import { ReportNameGenerator } from '../report/report-name-generator';
 import { ScanArguments } from '../scanner/scan-arguments';
 import { CrawlerCommandRunner } from './crawler-command-runner';
 
-// tslint:disable: no-empty
+// tslint:disable: no-object-literal-type-assertion  no-unsafe-any
+
 describe('CrawlerCommandRunner', () => {
+    const testUrl = 'http://localhost/';
+    const scanResult: ScanResults = {
+        summaryScanResults: {
+            failed: [],
+            passed: [],
+            unscannable: [],
+        },
+        errors: [],
+        scanMetadata: {
+            baseUrl: '',
+            basePageTitle: '',
+            userAgent: '',
+        },
+    };
+
+    let testInput: ScanArguments;
+    let crawlerOption: CrawlerRunOptions;
     let crawlerEntryPointMock: IMock<CrawlerEntryPoint>;
     let reportGeneratorMock: IMock<ReportGenerator>;
     let reportDiskWriterMock: IMock<ReportDiskWriter>;
     let reportNameGeneratorMock: IMock<ReportNameGenerator>;
+    let fsMock: IMock<typeof fs>;
     let testSubject: CrawlerCommandRunner;
-    // tslint:disable-next-line: no-http-string
-    const testUrl = 'http://www.bing.com';
-    const testInput: ScanArguments = { url: testUrl, output: '/users/xyz' };
-    // tslint:disable-next-line: mocha-no-side-effect-code
+
     beforeEach(() => {
+        testInput = { url: testUrl, output: './dir' };
+        crawlerOption = {
+            baseUrl: testInput.url,
+            localOutputDir: testInput.output,
+            existingUrls: undefined,
```

Comment:
why do these have to be defined? why not just cast the object instead?

Reply:
example?

Reply:
```

crawlerOption = {
    baseUrl: testInput.url,
    localOutputDir: testInput.output,
} as CrawlerRunOptions;
```
should do the trick.

Reply:
does not work, need to have explicit undefined

Reply:
Hm, I just tried it on my end, and it seemed to work I didn't even need the `as` casting. Hmmmm...